docs/_documentations/mdt-vsc-installinfo.md

```diff
@@ -22,7 +22,9 @@ The following images are pulled. These images together form the Codewind back en
 - `eclipse/codewind-pfe-amd64`
 
 ## Using Codewind
-When the installation is complete, the extension is ready to use, and you are prompted to open the Codewind workspace. For more information about using Codewind, see [Tutorial: Codewind for VS Code](mdt-vsc-tutorial.html).
+When the installation is complete, the extension is ready to use, and you are prompted to open the Codewind workspace.
+
+The `codewind-workspace` appears when Codewind creates the folder within your home directory to contain your projects. On Windows, you can find the workspace in the `C:\codewind-workspace` directory. You can open the `codewind-workspace` folder or a project within the workspace as your VS Code workspace. For more information, see [Getting started: Codewind for VS Code](mdt-vsc-getting-started.html).
```

Comment:
```
Codewind creates `~/codewind-workspace` to contain your projects. On Windows, you can find the workspace at `C:\codewind-workspace`.
```

We should mention this in eclipse too. use the same first sentence, change the second to:

```
You can open the `codewind-workspace` folder or a project within the workspace as your Eclipse workspace. For more information, see  <use the eclipse link>
```